WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -239,11 +251,83 @@ public void onCreate(Bundle savedInstanceState) {
             confirmInstallPluginForAutomatedTransfer();
         }
 
+        if (mIsShowingDomainCreditCheckProgress) {
+            showDomainCreditsCheckProgressDialog();
+        }
+
         if (mIsShowingAutomatedTransferProgress) {
             showAutomatedTransferProgressDialog();
         }
     }
 
+    @SuppressWarnings("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onPlansFetched(OnPlansFetched event) {
+        cancelDomainCreditsCheckProgressDialog();
+
+        if (event.isError()) {
+            AppLog.e(T.PLANS, PluginDetailActivity.class.getSimpleName() + ".onPlansFetched: "
```

Comment:
We may want to consider showing a snackbar so the user knows something went wrong, wdyt?

Reply:
I'm a bit afraid the errors aren't localized. We might want to use a string resource - if we don't know what happened even "unexpected error" is imho better than failing silently. Wdyt?